index.js

```diff
@@ -24,6 +25,30 @@ const modalBody = ReadElement('.modal-layout');
 const wrapper = document.querySelector('.wrapper');
 const wrapperVisible = wrapper.getAttribute('data-visible');
 
+const formObject = {
+  name: '',
+  email: '',
+  message: '',
+};
+
+function formFunc() {
+  formObject.name = inputName.value;
+  formObject.email = inputEmail.value;
+  formObject.message = inputMessage.value;
+  const stringForm = JSON.stringify(formObject);
+  localStorage.setItem('form', stringForm);
+}
+
+const localFormObject = JSON.parse(localStorage.getItem('form'));
+
+window.onload = () => {
+  if (localFormObject) {
+    inputName.value = localFormObject.name;
+    inputEmail.value = localFormObject.email;
+    inputMessage.value = localFormObject.message;
+  }
+};
+
```

Comment:
- [ ] I can see you managed to implement some requested changes by the previous review good job 👍 but still there is something you missed out, as required in the previous feedback consider implementing the below requirement 
> After filling the form, and not submitting it, the entered values should be saved in the browser.
- there it means as I fill out the form the data should be saved in the local storage immediately without any other action such as reloading, submitting,...... kindly consider implementing that this will make your project be on a high standard as it will meet all the requirements

Reply:
Thanks, it has been resolved